utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/extractor/JsonToObject.scala

```diff
@@ -31,9 +31,10 @@ object JsonToObject {
       extract[JsonObject](
         _.asObject,
         jo => TypedMap(
-          elementType.map { case (jsonField, jsonDef) =>
-            jsonField -> jo(jsonField).map(JsonToObject(_, jsonDef)).getOrElse(nullOrError(jsonField))
+          elementType.collect {
```

Comment:
1. Why you replaced map with collect that matches everything? (which is just a map)
2. It is possible to do this filter in collect `case ... if ...`? 
3. Current "going through type" way causes that additional properties are removed in simple source -> sink scenario. I'm wondering if it is ok?

Reply:
1. i can restore map
2. no because we need to check is evaluated value is null
3. only not requried properties that are null are removed - I think it's ok

Reply:
1. please restore
2. ok
3. I mean someone has "dynamic" (json-way) interface - it takes any document on input, filter by one, defined field, and send this document on output without modification. Current approach will remove all fields and left only one used in filtering

Reply:
1. done
3. we will change approach to iterate over incoming fields instead of input schema in separete task